zio/zjsonio/reader.go

```diff
@@ -48,7 +49,7 @@ func (r *Reader) Read() (*zng.Record, error) {
 		return nil, e(err)
 	}
 	// remove newline
-	line = line[:len(line)-1]
+	line = bytes.TrimSpace(line)
 	var v Record
 	err = json.Unmarshal(line, &v)
```

Comment:
json.Unmarshal ignores whitespace between tokens, so there's no need to fiddle with line like this.

Reply:
Ah, good point. So I can just remove the original `line = line[:len(line)-1]` then.